README.md

```diff
@@ -20,6 +20,8 @@ In the `pom.xml` file include the following:
   <version>1.5.0</version>
   <configuration>
     <mutationEngine>descartes</mutationEngine>
+    <!-- change the following into testng if that is your test framework -->
+    <testPlugin>junit</testPlugin>
```

Comment:
This is the option by default, so why putting it there?
In any case I would write the following
```
<!-- Add the following line if the test framework is TestNG -->
<testPlugin>junit</testPlugin>
```
Keep in mind that this is in fact an option for PIT. It has nothing to do with Descartes. PIT's configuration is very well documented in their website. I would prefer to add a small message saying that for more configuration options visit the PIT's website or something like that.

Reply:
Hi @oscarlvp , yes, that makes sense. I will remove this option and add something as you suggest.